src/main/java/com/uci/inbound/incoming/CampaignController.java

```diff
@@ -139,7 +139,7 @@ private void sendEventToKafka(XMessage xmsg) {
         } catch (JAXBException e) {
             kafkaProducer.send(topicFailure, "Start request for bot.");
         }
-        kafkaProducer.send(topicSuccess, xmessage);
+        kafkaProducer.send(campaign, xmessage);
```

Comment:
Can we have a more descriptive variable name here? `campaign` does not really describe the purpose for which it is used.